userspace/libscap/engine/gvisor/gvisor.h

```diff
@@ -168,6 +171,8 @@ class engine {
     std::string m_root_path;
     std::string m_trace_session_path;
 
+    scap_stats m_stats;
```

Comment:
nit: there's one tiny thing that I'm not completely happy with about this stats structure, it's the fact that it is not always updated. the `m_drops` is only updated if the `get_stats()` function is called. To make the invariant hold I would suggest the following: only save three fields in the class, that would be the number of drops on gvisor side (perhaps with a descriptive name like `m_stat_drops_gvisor` so we remember what it its), the number of drops because of parsing errors and the total number of events processed. This way we can fill only the relevant fields in the stats structure when requested and we don't need to maintain an `m_drops` that isn't updated and a bunch of zeroes for fields that are not relevant for gvisor.